iModelCore/ECDb/Tests/NonPublished/ECSqlStatementTests.cpp

```diff
@@ -169,6 +169,16 @@ TEST_F(ECSqlStatementTestFixture, NestedValConstructor) {
 /*---------------------------------------------------------------------------------**//**
 * @bsimethod
 +---------------+---------------+---------------+---------------+---------------+------*/
+TEST_F(ECSqlStatementTestFixture, RowValConstructor) {
+    ASSERT_EQ(DbResult::BE_SQLITE_OK, SetupECDb("RowValConstructor.ecdb"));
+
+    ECSqlStatement stmt;
+    ASSERT_EQ(ECSqlStatus::Success, stmt.Prepare(m_ecdb,"SELECT * FROM (VALUES(1,2), VALUES(3,4))"));

```

Comment:
This query in plain sqlite will not parse, instead, the sqlite syntax would be
```sql
SELECT * FROM (VALUES(1,2), (3,4))
```

Also, the first query returns row (1,2) first and then row(3,4) while the native sql below returns the reversed order. (3,4 first, then 1,2)

Reply:
I've fixed the order of returned values in the generated native SQL.

Reply:
Updated grammar parser rule to support `SELECT * FROM (VALUES(1,2), (3,4))` instead of `SELECT * FROM (VALUES(1,2), VALUES(3,4))`